js/data.json

```diff
@@ -1471,5 +1486,10 @@ osts = [
 		"link": "https://www.dropbox.com/s/adhvu2r1646c2j0/Gohan%20ya-san%20Megumi.mp3?dl=1",
 		"name": "Amaama to Inazuma OST - Gohan ya-san Megumi",
 		"img": "https://www.dropbox.com/s/suvtvzcdm5rptxy/amaama%204.jpg?dl=1"
+	},
+	{
+		"link": "https://www.dropbox.com/s/obccd111g0a4ho3/28%20-%20Samidare.mp3?dl=0",
+		"name": "Naruto OST - Samidare",
+		"img": "https://www.dropbox.com/s/c36rmzzdmkvpw54/jiraya_img.jfif?dl=0"
```

Comment:
This image seems invalid, .jfif format?

Reply:
:(

Reply:
my bad to have you go through this so many times. I'll make sure I won't repeat myself in future contributions.

Reply:
Lol no worries, it's alright 😄